docs/src/reconciling-local-changes.md

```diff
@@ -0,0 +1,234 @@
+stage, it will fail early when it cannot reach the server. 
+
+To reconcile this change in `config.yml` simply delete the entry under
+`instances` corresponding to `node-2`. It will look something like this:
+
+```yaml
+- Name: node-2
+  public_ip: 44.201.93.236
+  private_ip: 172.31.71.186
+  location: a
+  node: 2
+  role:
+  - bdr
+  - pgd-proxy
+  vars:
+    bdr_child_group: a_subgroup
+    bdr_node_options:
+      route_priority: 100
+```
+
+You can now manage this node as usual using TPA. However, the original
+cluster still has metadata that refers to `node-2` so to complete
+reconciliation it is recommended to run the following SQL on each node
+to remove the metadata. *This step is essential if you wish to add a
+node of the same name in future.*
+
+```sql
+select * from bdr.drop_node('node-2');
+```
+
+!!! Note 
+If you wish to join the original `node-2` back to the cluster after
+removing it in this way, you can do so simply by restoring the deleted
+lines of `config.yml` but you must ensure that `select * from
+bdr.drop_node('node-2');` has been run on this node and that the PGDATA
+directory has been deleted.
+!!!
+
+### Example: changing the superuser password
+TPA automatically generates a password for the superuser which you may
+view using `tpaexec show-password <cluster> <superuser-name>`. If you
+change the password manually (for example using the `/password` command
+in psql) you will find that after `tpaexec deploy` is next run, the
+password has reverted to the one set by TPA. To make the change through
+TPA, and therefore make it persist across runs of `tpaexec deploy`, you
+must use the command `tpaexec store-password <cluster> <superuser-name>`
+to specify the password, then run `tpaexec deploy`. This also applies to
+any other user created through TPA.
+
+### Example: adding or removing an extension
+A simple single-node cluster can be deployed with the following
+`config.yml`.
+
+```yaml
+---
+architecture: M1
+cluster_name: singlenode
+
+cluster_vars:
+  postgres_flavour: postgresql
+  postgres_version: '15'
+  preferred_python_version: python3
+  tpa_2q_repositories: []
+
+instance_defaults:
+  image: tpa/debian:11
+  platform: docker
+  vars:
+    ansible_user: root
+
+instances:
+- Name: nodeone
+  node: 1
+  role:
+  - primary
+```
+You may manually add the pgvector extension by connecting to the node
+and running `apt install postgresql-15-pgvector` then executing the
+following SQL command: `CREATE EXTENSION vector;`. This will not cause
+any operational issues, beyond the fact that `config.yml` no longer
+describes the cluster as fully as it did previously. However, it is
+advisable to reconcile `config.yml` (or indeed simply use TPA to add the
+extension in the first place) by adding the following cluster variables. 
+
+```yaml
+cluster_vars:
+  ...  
+  packages:
+   common:
+   - postgresql-15-pgvector
+  postgres_extensions:
+  - vector
```

Comment:
```suggestion
cluster_vars:
  ...  
  extra_postgres_packages:
   common:
   - postgresql-15-pgvector
  postgres_extensions:
  - vector
```

Reply:
Packages, such as extensions, top up an existing posgres installation, and therefore these packages should be installed after Postgres has already been installed. TPA has `extra_postgres_packages` for precisely the same kind of scenario. See `postgres_installation_method_pkg.md` for details